lib/components/viewers/pattern-row.tsx

```diff
@@ -111,7 +97,7 @@ class PatternRow extends Component<Props, State> {
                 />
               </div>
             </strong>
-            <span>{pattern.headsign}</span>
+            <span title={pattern.headsign}>{pattern.headsign}</span>
```

Comment:
I forget if we discussed this already but I don't think it's good practice to have the `title` contain the exact same content as the element itself!

Reply:
In this case I think it's useful when often we're seeing gtfs where the shortname or headsign significantly overflows the container! A stopgap so users still have access to the info